azure-pipelines.yml

```diff
@@ -116,16 +144,16 @@ jobs:
       frameworks: # for private jobs we want to benchmark .NET Core 3.0 only
         - netcoreapp3.0
         
-# Ubuntu 1604 x64 micro benchmarks, public correctness job
+# Ubuntu 1804 x64 micro benchmarks, public correctness job
 - ${{ if eq(variables['System.TeamProject'], 'public') }}:
   - template: /eng/performance/benchmark_jobs.yml
     parameters:
       osName: ubuntu
-      osVersion: 1604
+      osVersion: 1804
       kind: micro
       architecture: x64
       pool: Hosted Ubuntu 1604
-      queue: Ubuntu.1604.Amd64.Open
+      queue: Ubuntu.1804.Amd64.Open
```

Comment:
awesome! I am super happy to see the Ubuntu 18.04 being added to the list of tracked OSes!

/cc @tannergooding @danmosemsft ft

Reply:
A few questions:
* What are the plans, if any, for testing interim builds, like Ubuntu 19.04 or 19.10 when it comes out?
* Are there any goals at getting onto new LTS builds (e.g. 20.04) sooner after they come out?
* Given that there are 2-3 LTS builds out at a given time, what are the plans for the lifetime of the perf machines for each version?

Reply:
For perf, it's a bit of a challenge to move to new OSes. @billwert would probably be the best person to talk about in terms of future planning. I would guess that we will only run perf on LTS builds, but it requires us to split our pools, and we have limited hardware. So when we move to a new LTS, we will lose support for previous versions. We are just now moving from 1604 to 1804, and 1804 has been out for over a year.

Reply:
yeah, it would be my goal to run only LTSes, I think. At some point in the future perhaps we'll have an imaging story and will have more flexibility. Right now the OS is installed on bare metal, so shifting is a big deal. It also means that, as Michelle says, if we tried to run 2 LTSes at once we'd have to cut the pool in half. I would rather not do that. In the future we'll be able to respond more quickly. I think we'll pick up new LTSes when a branch of the product starts supporting that LTS. (So for 20.04, .NET Core 5 next year?)

There is a complication to this that older products may not work on newer LTS versions, so our ability to baseline things properly may be impacted. Need to think about that a bit.